### ISSUE TITLE
client stuck locating server - docker

### ISSUE THREAD

**Issue by IvanTurgenev** [codehz/ElementZero#6]
everything seems fine server side but client is stuck locating server, same client works perfectly with https://hub.docker.com/r/roemer/bedrock-server
```
ubuntu@user:~$ sudo docker run --name=mcpemod  -ti -p 19132:19132/udp -v /home/user/mcpemod:/data codehz/wine:bdlauncher-runtime
Injected!
HealthCheck 0000000140000000 0000000140054870
V [Base] (Mod::Base:145) Current thread id: 9
I [Base] (Mod::Base:146) Base mod loaded, setting up CtrlC handler...
V [LogDatabase] (Mod::Base:163) initialized (location log.db)
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\Audit.dll
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\BasicAntiCheat.dll
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\Blacklist.dll
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\CommandSupport.dll
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\DebugTool.dll
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\Economy.dll
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\Essentials.dll
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\NoEncryption.dll
V [ModLoader] (Mod::Base:87) Loaded Z:\data\Mods\SpawnProtection.dll
I [ModLoader] (Mod::Base:135) Init BasicAntiCheat
I [ModLoader] (Mod::Base:135) Init CommandSupport
I [ModLoader] (Mod::Base:135) Init DebugTool
I [ModLoader] (Mod::Base:135) Init NoEncryption
I [ModLoader] (Mod::Base:135) Init Audit
I [ModLoader] (Mod::Base:135) Init Blacklist
V [Blacklist] (Mod::Blacklist:25) Initialize database blacklist.db
I [ModLoader] (Mod::Base:135) Init Economy
I [ModLoader] (Mod::Base:135) Init Essentials
I [ModLoader] (Mod::Base:135) Init SpawnProtection
I [SERVER] (DedicatedServer::initializeLogging:166) setting up server logging...
I [SERVER] (DedicatedServer::start:217) Starting Server
I [SERVER] (DedicatedServer::start:218) Version 1.14.32.1 with element-0 (200310)
I [SERVER] (DedicatedServer::start:219) Session ID 40356aea-d828-4d78-9be5-3a78f9c61bb3
I [SERVER] (DedicatedServer::start:231) Level Name: Bedrock level
I [SERVER] (DedicatedServer::start:256) Game mode: 0 Survival
I [SERVER] (DedicatedServer::start:258) Difficulty: 1 EASY
I [DATABASE] (DBStorage::DBStorage:183) opening worlds/Bedrock level/db
I [NETWORK] (RakPeerHelper::LogIPSupport:130) IPv4 supported, port: 19132
I [NETWORK] (RakPeerHelper::LogIPSupport:141) IPv6 not supported
I [SERVER] (DedicatedServer::start:529) Server started.
^RallallowI [SERVER] (ServerNetworkHandler::_onClientAuthenticated:1331) Player connected: PLAYER, xuid: 2533274971772643
V [PlayerDB] (Mod::Base:126) PLAYER joined (from 186.119.66.111|61153)
```

**Comment by codehz** [comment]
try connect it with different devices... it seems to be a upstream bug (bds windows specified

**Comment by IvanTurgenev** [comment]
Thanks for the reply tried on noxplayer osx, and an Android 10 device same
result

**Comment by codehz** [comment]
You can try to use `--net host ` and change the port number in config

**Comment by IvanTurgenev** [comment]
it worked

**Comment by IvanTurgenev** [comment]
what are the docker issues with the user id ?

**Comment by IvanTurgenev** [comment]
> 
> 
> it worked the host thing

on minecraft microsoft store version, cant get it to work on android, though it worked on local network with android, not when it was on a vps

**Comment by IvanTurgenev** [comment]
it seems it doesnt work with android when hosted away it works on lan though

**Comment by IvanTurgenev** [comment]
looks like this is it? idk
https://bugs.mojang.com/projects/BDS/issues/BDS-1094?filter=allopenissues

**Comment by IvanTurgenev** [comment]
this has been fixed by moving from vps and updating to 1.4.60.5

**Comment by IvanTurgenev** [comment]
it seems the cause was permissions files must be own by root
`sudo chown -R root  ./mcpe`
` sudo chmod -R ug+rw ./mcpe`
